frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -48,45 +47,92 @@ const Navbar = ({children}) => {
+          }}
+        >
+          Sign Up
+        </button>
       </p>
     </>
   );
 
   return (
     <>
-      <nav className="top-0 left-0 right-0 w-full">
+      <a className="top-0 left-0 right-0 w-full">
```

Biome (1.9.4)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)